ouroboros-consensus/ouroboros-consensus.cabal

```diff
@@ -322,7 +322,7 @@ library
     , sop-core                     >=0.5      && <0.6
     , streaming
     , strict-stm                   ^>=1.1
-    , text                         ^>=2.0
+    , text                         >=1.2
```

Comment:
I'm not sure about the transitive ramifications either to be honest. I found this to be causing a conflict with some upstream hackage library dependency that had a `^>=1.2`  and am running a trial of whether the entire chain up to `api` could build, since I still haven't gotten over the SRP usage yet.